src/components/results/loadflow/load-flow-result-utils.ts

```diff
@@ -415,22 +414,38 @@ export const loadFlowResultColumnsDefinition = (
         }),
         makeAgGridCustomHeaderColumn({
             headerName: intl.formatMessage({ id: 'slackBusId' }),
-            field: 'slackBusId',
-            sortProps,
+            field: 'id',
             filterProps,
             filterParams: textFilterParams,
         }),
         makeAgGridCustomHeaderColumn({
             headerName: intl.formatMessage({
                 id: 'slackBusActivePowerMismatch',
             }),
-            field: 'slackBusActivePowerMismatch',
+            field: 'activePowerMismatch',
             numeric: true,
             fractionDigits: 2,
-            sortProps,
             filterProps,
             filterParams: numericFilterParams,
-            cellRenderer: numberRenderer,
```

Comment:
If it's no longer used, we should remove numberRenderer from the props

Reply:
In fact I should keep it.